.github/actions/build-and-cache-rust-tool/action.yml

```diff
@@ -68,7 +68,7 @@ runs:
       uses: actions/upload-artifact@v4
       with:
         name: ${{ inputs.artifact-upload-name }}
-        path: ~/.cargo/bin/${{ inputs.artifact-artifact-bin-name }}
+        path: ~/.cargo/bin/${{ inputs.artifact-bin-name }}
```

Comment:
Whopsie ... this lead to uploading the whole `~/.cargo/bin/` folder instead of just `~/.cargo/bin/cargo-nextest` since a non-existing variable is obviously equal to an empty one 🤬

Reply:
hehehe